csrankings-c.csv

```diff
@@ -62,6 +62,7 @@ Candace L. Sidner,Worcester Polytechnic Institute,http://www.wpi.edu/academics/f
 Candido Cabo,CUNY,https://www.gc.cuny.edu/Page-Elements/Academics-Research-Centers-Initiatives/Doctoral-Programs/Computer-Science/Faculty-Bios/Candido-Cabo,pvFqstUAAAAJ
 Candy L. Sidner,Worcester Polytechnic Institute,http://www.wpi.edu/academics/facultydir/cls.html,j8ULfMsAAAAJ
 Cang Ye,Virginia Commonwealth University,https://egr.vcu.edu/directory/cangye,3kvDgyMAAAAJ
+Cao Lei,University of Arizona,https://www.cs.arizona.edu/person/lei-cao,xiSS1ZIAAAAJ

```

Comment:
Needs disambiguation suffix (0004?) - https://dblp.org/pid/17/6060-4.html

Reply:
Right, that is the DBLP page. Where can I put the suffix 0004? I did not see a place to put DBLP related info?